python/python/glide/async_commands/transaction.py

```diff
@@ -981,6 +984,108 @@ def zcount(
         """
         self.append_command(RequestType.Zcount, [key, min_score.value, max_score.value])
 
+    def zrange(
```

Comment:
Lets move all of the helper classes (ScoreBoundary etc) and the _create_zrange_args function to a new file sorted_set.py. We need to start splitting the commands into separate files, it's getting too large